### PR TITLE
fix collections in various entities.

### PR DESCRIPTION
**Pull request by finsharp**
After checking the metadata, the fields in the change-set should be collections.

### CODE REVIEW

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[b503566](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4213/commits/b503566ae6fa17f99fd89c52d13f0e21e05f15de)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/directorysetting-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/broken-collections/api-reference/beta/api/directorysetting-update.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/governancerolesetting-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/broken-collections/api-reference/beta/api/governancerolesetting-update.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/orgcontact-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/broken-collections/api-reference/beta/api/orgcontact-update.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/permission-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/broken-collections/api-reference/beta/api/permission-update.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/privilegedrolesettings-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/broken-collections/api-reference/beta/api/privilegedrolesettings-update.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/range-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/broken-collections/api-reference/beta/api/range-update.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C20%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903202001274048-4213%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=nlddgJdtofN%2FhgIEPqF2Ok03vbCwlovrib3UbneCJOw%3D&st=2019-03-20T19%3A59%3A51Z&se=2019-04-20T20%3A04%3A51Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api-reference/beta/api/directorysetting-update.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ In the request body, supply the values for relevant fields that should be update
 
 | Property	   | Type	|Description|
 |:---------------|:--------|:----------|
-| values | settingValue | The updated set of values.  NOTE: You must supply the entire collection set. You cannot update a single set of values. |
+| values | settingValue collection | The updated set of values.  NOTE: You must supply the entire collection set. You cannot update a single set of values. |
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/directorysetting-update.md]: Can you please add a link to settingValue?

Comment (finsharp) [reply api-reference/beta/api/directorysetting-update.md]: Added the link.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[c9c9a79](https://github.com/microsoftgraph/microsoft-graph-docs/pull/4213/commits/c9c9a7939ae07cb0bff4e1050601ed0c2deb5644)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/directorysetting-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/broken-collections/api-reference/beta/api/directorysetting-update.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/governancerolesetting-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/broken-collections/api-reference/beta/api/governancerolesetting-update.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/orgcontact-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/broken-collections/api-reference/beta/api/orgcontact-update.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/permission-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/broken-collections/api-reference/beta/api/permission-update.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/privilegedrolesettings-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/broken-collections/api-reference/beta/api/privilegedrolesettings-update.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/range-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/broken-collections/api-reference/beta/api/range-update.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C20%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903202140126830-4213%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=%2BvWPavvtPHaNoq447tR4p81Hwc8TcisA8JQd15gx5OI%3D&st=2019-03-20T21%3A38%3A36Z&se=2019-04-20T21%3A43%3A36Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by finsharp** [comment]
@angelgolfer-ms I am not yet authorized to merge, could you merge this for me

**Pull request merged**